package.json

```diff
@@ -84,6 +79,14 @@
     "@storybook/addon-links": "^3.4.8",
     "@storybook/addons": "^3.4.8",
     "@storybook/react": "^3.4.8",
+    "@types/axios": "^0.14.0",
+    "@types/lodash": "^4.14.116",
+    "@types/qs": "^6.5.1",
+    "@types/react": "^16.4.9",
+    "@types/react-currency-formatter": "^1.1.0",
+    "@types/react-dom": "^16.0.7",
+    "@types/react-router-dom": "^4.3.0",
+    "@types/react-select": "^2.0.2",
```

Comment:
I think we moved these into `dependencies` because we were having issues with our Heroku production builds

Reply:
Relevant issue https://github.com/rails/webpacker/issues/117

Reply:
ah i see. i'll move them back. that might be why the build is failing